src/biotite/structure/bonds.pyx

```diff
@@ -1645,12 +1647,37 @@ def connect_via_residue_names(atoms, bint inter_residue=True,
                 res_names[curr_start_i], {}
             )
 
+        # Check if we should use alternative atom names
         atom_names_in_res = atom_names[curr_start_i : next_start_i]
+        std_atom_ids = get_from_ccd(
+            "chem_comp_atom", 
+            res_names[curr_start_i], 
+            "atom_id"
+        ) 
```

Comment:
`get_from_ccd()` returns a `BinaryCIFColumn` which needs to be converted into a NumPy array.

```suggestion
        std_atom_ids = get_from_ccd(
            "chem_comp_atom", 
            res_names[curr_start_i], 
            "atom_id"
        )
        std_atom_ids = std_atom_ids.as_array() if std_atom_ids is not None else None
```